dre_probe_matcher.py

```diff
@@ -250,6 +257,27 @@ def __del__(self):
             self.output_adept.close()
 
 
+    def should_file_run(self, filename):
+        '''
+        Looks to see if the input file already has a matching output file. If it does and:
+        1. We are not running alignment OR
+        2. We are running alignment and alignment has already been calculated for this file
```

Comment:
It looks like this sentence wasn't ended-- as if it needs, "then return False".  But doesn't it also depend on `RUN_ALL`?

Reply:
great catch. Fixed!